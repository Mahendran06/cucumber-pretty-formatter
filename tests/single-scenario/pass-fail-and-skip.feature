Feature: Pass fail and skip scenario
  In order to test event stream formatters
  As formatter
  I need to handle scenario with one failing step after passing and another should skip

  Scenario: Scenario
    Given passing
    When failing
    Then passing
