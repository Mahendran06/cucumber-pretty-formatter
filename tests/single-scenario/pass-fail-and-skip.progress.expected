.F_                                                                    3

--- Failed steps:
    When failing # tests/single-scenario/pass-fail-and-skip.feature:8
      Error: step failed error

1 scenarios (1 failed)
3 steps (1 passed, 1 failed, 1 skipped)
0s (0B)
